tests/test_sqlRegistry.py

```diff
@@ -376,6 +377,17 @@ def testBasicTransaction(self):
         # Or the DataUnit entries
         self.assertIsNone(registry.findDataUnitEntry("Camera", {"camera": "DummyCam"}))
 
+    def testGetRegion(self):
+        registry = Registry.fromConfig(self.butlerConfig, create=True)
+        region = lsst.sphgeom.ConvexPolygon((lsst.sphgeom.UnitVector3d(1, 0, 0),
+                                             lsst.sphgeom.UnitVector3d(0, 1, 0),
+                                             lsst.sphgeom.UnitVector3d(0, 0, 1)))
+        registry.addDataUnitEntry("SkyMap", {"skymap": "DummySkyMap", "sha1": bytes()})
```

Comment:
I've just noticed we are insisting on this being a `sha1`. Would it generically be better to call this `hash` or something?